docs/tutorials/analysis/3D/event_sampling.ipynb

```diff
@@ -75,11 +75,15 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Remove and just use the builtin <code>ExpDecayTemporalModel</code> here...
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>